test/univariate/continuous/chisq.jl

```diff
@@ -1,2 +1,8 @@
-test_cgf(Chisq(1), (0.49, -1, -100, -1f6))
-test_cgf(Chisq(3), (0.49, -1, -100, -1f6))
+
+@testset "Chisq" begin
+    test_cgf(Chisq(1), (0.49, -1, -100, -1.0f6))
+    test_cgf(Chisq(3), (0.49, -1, -100, -1.0f6))
+
+    @test rand(Chisq(1.0)) isa Float64
+    @test rand(Chisq(1.0f0)) isa Float32
```

Comment:
This just happens to be the case but IMO it is not really something that we have (or should) generalize in this PR.

Reply:
Okay then let me axe this.